docs/application/dotnet/guides/multimedia/audio.md

```diff
@@ -140,8 +140,34 @@ To query audio device information:
         > **Note**   
 		> The initial running state of the connected device is `false`, which means the connected device is not running.
 
+-   More functionality for AudioDeviceType.UsbAudio and AudioDeviceIoDirection.Output device, you can utilize more functionality with the following methods of the [Tizen.Multimedia.AudioDevice](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.AudioDevice.html) class:
+    -   To get the supported sample formats and to set the specific sample format:
 
+        1.  Retrieve the list of the currently supported sample formats with the `GetSupportedSampleFormats()` method:
 
+            ```
```

Comment:
Global comment:
Please add language identifier to the code blocks. For more information, check this: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#inline-code-blocks-with-language-identifier

Reply:
Okay